test-curves/src/bls12_381/fq2.rs

```diff
@@ -22,8 +22,24 @@ impl Fp2Config for Fq2Config {
     ];
 
     #[inline(always)]
-    fn mul_fp_by_nonresidue(fp: &Self::Fp) -> Self::Fp {
-        -(*fp)
+    fn mul_fp_by_nonresidue_in_place(fp: &mut Self::Fp) -> &mut Self::Fp {
```

Comment:
Most of our `in_place` functions do not have a return value. Should we keep the return value here?

Reply:
I can check if we use it anywhere. We're kind of inconsistent about it elsewhere.

Reply:
We do use it to avoid some temporaries, so I will leave it as is.